src/test/java/blackjack/domain/DeckTest.java

```diff
@@ -0,0 +1,51 @@
+package blackjack.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class DeckTest {
+
+    @DisplayName("덱에서 카드를 한 장 뽑는다")
+    @Test
+    public void draw() {
+        Deck deck = new Deck(List.of(CardFixture.diamond3()), Collections::shuffle);
+
+        assertThat(deck.draw()).isEqualTo(new Card(CardSuit.DIAMOND, CardNumber.THREE));
+    }
```

Comment:
```suggestion
class DeckTest {

    @DisplayName("덱에서 카드를 한 장 뽑는다")
    @Test
    void draw() {
        Deck deck = new Deck(List.of(CardFixture.diamond3()), Collections::shuffle);

        assertThat(deck.draw()).isEqualTo(new Card(CardSuit.DIAMOND, CardNumber.THREE));
    }
```
jUnit5 부터는 테스트 클래스와 메서드에 public 접근제한자를 요구하지 않아요. 그러니 필수가 아니라면 빼는게 어떨까요?

Reply:
커스텀 템플릿을 사용중이었는데 테스트 메서드에 public이 붙어있었네요. 싹 지우고 수정하겠습니다.